### DISCUSSION TITLE
Sponsors button in read me profiles for non GitHub sponsored

### DISCUSSION THREAD

**Discussion by Fluffy728** [community/community#127490] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I want to link to an external  link for donating on my profile without the power of GitHub sponsors. How! I know you can do this with code areas but not profiles, maybe we should add it?

**Comment by Fluffy728** [comment]
Ok but how would we do this

**Comment by Fluffy728** [comment]
There’s a workaround by adding a link to your profile but we need to add this as a feature

**Comment by ebndev** [comment, marked as answer]
Hi @Fluffy728, thanks for being a part of the GitHub Community!

Unfortunately there is not a way to do this other than adding a link like you did or creating a section in your profile's README. I don't see that you have one yet - you're more than welcome to create one and provide more information for others visiting your profile 😄 

If you do reconsider GitHub Sponsors, here's more information on [Setting Up Sponsors](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account).

Best of luck!

**Reply by Fluffy728** [reply]
Or you could just link a link in profile but maybe later

**Comment by Fluffy728** [comment]
Ok I could consider GitHub sposnsored and if I don’t just link a link in my profile
